### PR TITLE
Task 1 - CSV To Json

### PR DESCRIPTION
**Pull request by JLodha**
__1__ 

#### Short description of what this resolves:
Completed Task 1
The file in Submissions/JLodha/Task_1/ contains csvtojson.py ,when run converts the csv to json
The outputs are already included in the folder.

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by deepakDOTexe** [comment]
@JLodha  Take a look at this PR and create one of yours just like this. You can talk to the creator of this PR in case you are facing any issues.
https://github.com/opencodeiiita/Monty_Python_Class/pull/7/files

**Comment by JLodha** [comment]
Sir Updated!
Please have a look.
Thanks

File: Submissions/JLodha/Task_1/csvtojson.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import json
+
+csvfile = open('movies.csv', encoding = "utf8",errors = 'ignore')
+fields = ['movieId','title','genres']
+reader = csv.DictReader(csvfile, fieldnames = fields)
+datalist = []
+movie_genre= 'Comedy'
+for row in reader:
+    newrow = {'movieId' : row['movieId'], 'title':row['title'], 'genres': row['genres']}
+    if(newrow['genres'] == movie_genre):
```
Comment (deepakDOTexe) [inline Submissions/JLodha/Task_1/csvtojson.py]: There can be movies whose genre contains other genres also apart from only 'Comedy' such as 'Drama' or 'Action' so make your code more flexible.

Comment (JLodha) [reply Submissions/JLodha/Task_1/csvtojson.py]: Updated
Please have a look at it
Thanks

**Review by deepakDOTexe — APPROVED** [review]
PR can be merged now

**Pull request merged**